src/pages/Blogs/singleBlog.tsx

```diff
@@ -44,6 +44,13 @@ const SingleBlogView = () => {
       dispatch(getBlogById(id));
   }, [dispatch]);
 
+  const handleDeleteBlog = async (id) => {
+    
+      await dispatch(deleteBlogAction(id));
+      window.location.href = "/blogs#/applicant/blogs";
+    
+  }
```

Comment:
Please improve this implementation, for instance if there is any error this will be wrong

Reply:
Error handling is the redux actions plus the toasters